content/en/extensions/migrations/endpoint_migration_for_apps.md

```diff
@@ -0,0 +1,165 @@
+---
+title: "Endpoint migration for apps"
+description: "Here we will describe how to perform the migration task from endpoints to packages and services for Slingr apps. For App Developers"
+date: 2020-11-16T13:59:39+01:00
+lastmod: 2020-11-16T13:59:39+01:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "migrations"
+toc: true
+weight: 14
+---
+
+Endpoint Migration for Apps
+
+Technical documentation
+
+
+## Summary
+
+Here we will describe how to perform the migration task from endpoints to packages and services for Slingr apps. For App Developers
+
+For this example case we will use the application "test1" in its "dev" environment: <https://test1.slingrs.io/dev/builder/> . And the endpoint we will migrate will be "PandaDoc" with version v3.8.4.
+
+## Requirements
+
+Before we can start the migration task we need:
+
+- Have a Developer account on Slingr.
+
+- Have permissions to access the builder in the app to be migrated.
+
+To begin, first make sure that the package is registered in the Developer Portal. To do this go to <https://developer-portal.slingrs.io/#packages> . If the package exists you can continue, otherwise you must search for its repository and add it with the “**+ Create**” button.
+
+![register package](/images/vendor/extensions/register_package.png)
+
+Once verified, we can go to the Builder of the application to which the migration will be performed.
+
+The official endpoints have undergone a Standardization process. Therefore, the latest stable versions of these endpoints contain significant changes such as the addition of Flow Steps, changes in the way of referencing helpers (functions) and bug fixes.
+
+- Make sure you understand how the references to helpers have changed.
+
+_Note: the helper functions can typically be accessed similarly as is the name of the external service and the http operation to be performed. For example if you have the operation "_**_GET https\://api.pandadoc.com/documents/idOfTheDocument_**_" the reference to the function from an endpoint will be: "_**_app.endpoints.pandadoc.documents.get(idOfTheDocument)_**_". This helps to better identify which function is used and which external service is referenced in a more standard way in all endpoints._
+
+
+## Creating a Http Service
+
+1. Once inside, go, according to the App navigation panel, to: **Model -> Services** .
```

Comment:
I think we should update the path of where the services and extensions are located from this sprint.